EIPS/eip-4844.md

```diff
@@ -271,33 +275,48 @@ def point_evaluation_precompile(input: Bytes) -> Bytes:
 
 ### Gas price of blobs (Simplified version)
 
-For early draft implementations, we simply change `get_blob_gas(parent)` to always return `GAS_PER_BLOB`.
+___WARNING: This is only for testing___
 
-### Gas price update rule (Full version)
+For early draft implementations, we simply change `get_blob_gas(parent)` to always return `SIMPLE_GAS_PER_BLOB`.
 
-We propose a simple independent EIP-1559-style targeting rule to compute the gas cost of the transaction.
-We use the `excess_blobs` header field to store persistent data needed to compute the cost.
+### Gas accounting (Full version)
+
+We introduce data gas as a new type of gas. It is independent of normal gas and follows its own targeting rule, similar to EIP-1559.
```

Comment:
I _really_ dislike introducing a new resource separate from gas and calling it "data gas". The term "data gas" implies a subtype of gas—this isn't.

As a radical suggestion, I'd suggest calling this resource [teu](https://en.wikipedia.org/wiki/Twenty-foot_equivalent_unit).

I won't let this hold up the PR though.